tests/draft3/ref.json

```diff
@@ -207,17 +207,17 @@
             "definitions": {
                 "foo": {
                     "id": "http://localhost:1234/sibling_id/foo.json",
-                    "minimum": 2
+                    "type": "string"
```

Comment:
A more minimal change would be to fix the ranges of data used in the input, but using types is much less ambiguous, so 👍🏼

Reply:
first i tried by correcting ranges. and noticed that i got the test wrong. So decided to use type assertion to make it easier to understand the test